types/line-by-line/index.d.ts

```diff
@@ -5,12 +5,12 @@
 
 /// <reference types="node" />
 
-import { EventEmitter } from "events";
-import { Readable } from "stream";
+import { EventEmitter } from 'events';
+import { Readable } from 'stream';
 
 interface LineByLineReaderOptions {
     /** The encoding to use. */
-    encoding?: "ascii" | "base64" | "utf8";
+    encoding?: 'ascii' | 'utf8' | 'utf16le' | 'ucs2' | 'base64' | 'latin1' | 'binary' | 'hex';
```

Comment:
Documentation only states that `"ascii" | "base64" | "utf8"` are allowed. Could you please point to where the other values are permitted?

Reply:
Hi @armanio123 

encoding setting is used here:

```
		readStream = this._readStream ? this._readStream :
			fs.createReadStream(this._filepath, this._streamOptions);
```

[fs.createReadStream documentation states](https://nodejs.org/api/fs.html#fs_fs_createreadstream_path_options): "The encoding can be any one of those accepted by Buffer.", which in turn are [all the ones listed in this PR](https://nodejs.org/api/buffer.html#buffer_buffers_and_character_encodings).

Reply:
That makes sense, thanks.